postman/v1.0/Secure API Gateway v1.0 - Open Banking UK TPP Flows.postman_collection.json

```diff
@@ -281,7 +281,7 @@
 									}
 								},
 								"url": {
-									"raw": "https://{{IG-FQDN}}/jwkms/apiclient/issuecert",
+									"raw": "https://{{TRUSTEDDIR-FQDN}}/jwkms/apiclient/issuecert",
 									"protocol": "https",
 									"host": [
 										"{{IG-FQDN}}"
```

Comment:
I made this mistake before, I don't really understand the purpose of the "raw" field as the "host" below needs to be updated too, seems to duplicate data for no obvious reason.

Reply:
That is done now, thank you!